sdk/communication/Azure.Communication.Administration/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -24,6 +26,32 @@ public Sample2_PhoneNumberAdministrationClient(bool isAsync) : base(isAsync)
         {
         }
 
+        [Test]
+        public async Task CreatePhoneNumberWithTokenCredential()
```

Comment:
Is this really just "sample" or is it runnable test?

Reply:
It is a sample that can be used in our README.md but it is also runnable to ensure that the sample is always up to date with the latest code